package.json

```
@@ -3,11 +3,12 @@
   "version": "2.4.1",
   "description": "remark preset to configure remark-lint with settings for nodejs/node",
   "main": "index.js",
```

Not sure if this is required, but I think I had flipped this in my (non-working) branch
```suggestion
  "main": "./index.js",
```